channeld/channeld.c

```diff
@@ -539,6 +539,17 @@ static void handle_peer_splice_locked(struct peer *peer, const u8 *msg)
 	check_mutual_splice_locked(peer);
 }
 
+static void send_peer_my_alt_addr(struct peer *peer)
+{
+	/* Send my alt addr to peer db */
+	u8 *peer_msg = towire_peer_alt_addr(peer, peer->my_alt_addr);
+	peer_write(peer->pps, take(peer_msg));
+
+	/* We need the addrs in my own db too for later whitelist confirmation */
+	u8 *msg = towire_channeld_my_alt_addr(tmpctx, peer->my_alt_addr);
+	wire_sync_write(MASTER_FD, take(msg));
+}
```

Comment:
This is a very weird thing for channeld to do.  It's more gossipy than channel-related.  These days, this kind of thing is handled by connectd.